source/Irrlicht/COpenGLDriver.cpp

```diff
@@ -2309,7 +2332,20 @@ void COpenGLDriver::SAuxContext::updateNextState_pipelineAndRaster(const IGPURen
         static_cast<const COpenGLRenderpassIndependentPipeline*>(_pipeline)
     );
     if (_pipeline)
+    {
+        SOpenGLState::SGraphicsPipelineHash hash;
+        std::fill(hash.begin(), hash.end(), 0u);
+        nextState.usedShadersHash = hash;
```

Comment:
aren't you supposed to have a `!_pipeline`  in the if statement?

Reply:
yea fixed in commit yesterday